docs/how_it_works/devkit/pce.mdx

```diff
@@ -0,0 +1,73 @@
+# How Privacy Computation Engine Works
+
+## Configuring your Privacy Computation Engine
+
+You must first configure the Privacy Computation Engine in order to make the most of it.
+It needs to know about:
+- General information about your Organization and its' data processing policies (Equivalent to a RoPA - Record of Processing Activities)
+- Mappings of your data structures to [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
+- Few choices you have to make about when to authenticate the user (before or after they make a Privacy Request), and when to deliver automatic responses and when to wait for validation
+
+To Configure your Privacy Computation Engine prepare the following information:
+| Information | Format | Example|
+| :----: | :-----: | :--------|
+| Name and contact details of your Organization and its representative | `string` | `Blindnet Inc. hello@blindnet.io Vuk Janosevic`|
+| Identity and contact of a Data Protection Officer - if you are using blindnet devkit Privacy Request Builder, include the URL where you are hosting the interface | `string` | `Vuk Janosevic, www.blindnet.io/privacy-request-builder`|
+| If you are processing data on behalf of someone else (a Controller), then names, contact details of all such Controllers their representatives and the data protection officers | `string` | see above |
+| A link to your Privacy Policy | `URL` | `https://blindnet.io/privacy` |
+| A general description of the technical and organisational security measures referred to in Article 32 of GDPR | `string` | `We use administrative, technical, and physical safeguards to protect your personal data, taking into account the nature of the personal data and the processing, and the threats posed.`|
+| Countries where data servers are located (included those of the processors who are processing data on your behalf) | `string` | `France` |
+| A list of Retention Policies | [Retention Policies](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#retention-policy) | `data-categories`:`CONTACT`; `policy-type`:`NO-LONGER-THAN`, `duration`: `180 days`, `after`: `RELATIONSHIP-END` |
```

Comment:
I'd start by defining selectors (data category for each).
For each data category/selector, define list of provenances and list of retention policies.

Then define legal bases (type, name, description, privacy scope).

Having those 2, **Processing Records** can be inferred.

Reply:
I am OK with that as well. Feel free to change accordingly.

The question is really what is easier for the peson configuring the system to define? Is it easier to reason in terms of provenances, lgeal bases, retention policies (and have RoPA derived) or int erms of RoPA (and have the provenances, lgeal bases, retention policies derived).

I think indeed RoPA is a bit of an articial cosntruct.

Reply:
I made the change as you suggested. 

A mapping of database-field : selector is provided.

[Provenances](https://github.com/blindnet-io/product-management/blob/priv-patch/refs/schemas/priv/RFC-PRIV.md#provenance) and [Retention Policies](https://github.com/blindnet-io/product-management/blob/priv-patch/refs/schemas/priv/RFC-PRIV.md#retention-policy) already include Data Category/Selector in the new PRIV patch, so it is enought just to give a list of those objects.